controlplane/kubeadm/controllers/controller.go

```diff
@@ -254,6 +254,11 @@ func (r *KubeadmControlPlaneReconciler) reconcile(ctx context.Context, cluster *
 		return ctrl.Result{Requeue: true}, nil
 	}
 
+	// Ensure kubeadm role bindings for v1.18+
```

Comment:
Should we expand this comment to add more color, specifically reasons why this is needed?

Reply:
Added link to k/k commit